### DISCUSSION TITLE
Assignment II is out

### DISCUSSION THREAD

**Discussion by phschaer** [irgroup-classrooms/dis17-2021#24] [Announcements]
I have included the text for assignment II on the main page. We can talk about this on Friday. 

All the best

Philipp Schaer

**Comment by AnneSzyprons** [comment]
The text says that the concept paper should be uploaded on 2021-12-09. That would be Thursday. In the spreadsheet, assignment 2 is due on Friday, December 10. Has it been changed to Thursday?

**Reply by phschaer** [reply]
Fixed - The correct date is Friday, December 10! Sorry for the confusion. 

Please take the opportunity to talk about your ideas with Timo Breuer on Friday!
